internal/kafka/internal/workers/kafka_mgrs/preparing_kafkas_mgr.go

```diff
@@ -92,6 +92,7 @@ func (k *PreparingKafkaManager) handleKafkaRequestCreationError(kafkaRequest *db
 				return errors.Wrapf(updateErr, "Failed to update kafka %s in failed state. Kafka failed reason %s", kafkaRequest.ID, kafkaRequest.FailedReason)
 			}
 			metrics.UpdateKafkaRequestsStatusSinceCreatedMetric(constants2.KafkaRequestStatusFailed, kafkaRequest.ID, kafkaRequest.ClusterID, time.Since(kafkaRequest.CreatedAt))
+			metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusFailed, kafkaRequest.ID, kafkaRequest.ClusterID)
```

Comment:
Same as above, let's only update this if the status is not already `Failed`. This is called on every reconcile so it will get updated every few seconds.